apps/financial-aid/web-veita/src/components/NewMunicipalityModal/NewMunicipalityModal.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { useState } from 'react'
+import { Box, Input, Select, Text, Option } from '@island.is/island-ui/core'
+import { ActionModal } from '@island.is/financial-aid-web/veita/src/components'
+import { useMutation } from '@apollo/client'
+import { isEmailValid } from '@island.is/financial-aid/shared/lib'
+
+import { serviceCenters } from '@island.is/financial-aid/shared/data'
+import cn from 'classnames'
+import { MunicipalityMutation } from '@island.is/financial-aid-web/veita/graphql'
+
+interface Props {
+  isVisible: boolean
+  setIsVisible: React.Dispatch<React.SetStateAction<boolean>>
+  activeMunicipality?: number[]
+  onMunicipalityCreated: () => void
+}
+
+interface newMunicipalitModalState {
+  serviceCenter: Option
+  adminNationalId: string
+  adminName: string
+  adminEmail: string
+  hasError: boolean
+  hasSubmitError: boolean
+}
+
+const NewMunicipalityModal = ({
+  isVisible,
+  setIsVisible,
+  activeMunicipality,
+  onMunicipalityCreated,
+}: Props) => {
+  const selectServiceCenter = serviceCenters
+    .filter((el) => el.number !== 0 && !activeMunicipality?.includes(el.number))
+    .map((el) => {
+      return { label: el.name, value: el.number.toString() }
```

Comment:
Should label and value be municipalityName and municipalityCode instead?

Reply:
I am using this in select component, it takes props that allows me to show options and the options have to be label and value 😸

Reply:
Don't like it but makes sense 🤗